src/managers/KeybindManager.cpp

```diff
@@ -1202,12 +1202,13 @@ SDispatchResult CKeybindManager::changeworkspace(std::string args) {
 
     const static auto PWARPONWORKSPACECHANGE = CConfigValue<Hyprlang::INT>("cursor:warp_on_change_workspace");
 
-    if (*PWARPONWORKSPACECHANGE) {
+    if (*PWARPONWORKSPACECHANGE > 0) {
         auto PLAST     = pWorkspaceToChangeTo->getLastFocusedWindow();
         auto HLSurface = CWLSurface::fromResource(g_pSeatManager->state.pointerFocus.lock());
 
+        bool force = *PWARPONWORKSPACECHANGE == 2;
```

Comment:
why the variable lol just throw the expr into the func

Reply:
Indeed, done.